cypress/integration/templates_test.js

```diff
@@ -0,0 +1,45 @@
+describe("Templates Test", function() {
+    beforeEach(function() {
+      cy.visit("/");
+    });
+
+    it("can be accessed from the navigation tab", function() {
+        cy.openTemplatesTab();
+        cy.url().should("include", "/templates");
+    });
+
+    it("can be accessed from the main page", function() {
+        cy.contains("More queries here").click();
+        cy.url().should("include", "/templates");
+    });
+
+    it("can filter templates by search", function() {
+        cy.openTemplatesTab();
+        cy.url().should("include", "/templates");
+        cy.get('input[class="form-control input-lg"]').type("Organism{enter}",{delay:100});
```

Comment:
Did you try selecting this based on the *Filter by description* label? They're siblings so you might be able to do `cy.contains('Filter by description').next()`.